_data/gallery.yml

```diff
@@ -1,17 +1,23 @@
-- image: /media/gallery/1.jpg
-  title: A tranquil bay with grass and trees
+- image: /media/gallery/mtg.jpg
+  title: A castle in Minetest Game
 
-- image: /media/gallery/2.jpg
-  title: South pacific old growth forest
+- image: /media/gallery/rpg16.jpg
+  title: "[RPG-16](https://content.minetest.net/packages/Hugues%20Ross/rpg16/) texture pack"
 
-- image: /media/gallery/3.jpg
-  title: Castles in a large multiplayer server
+- image: /media/gallery/soothing32.jpg
+  title: "[Soothing32](https://content.minetest.net/packages/Zughy/soothing32/) texture pack"
 
-- image: /media/gallery/4.jpg
-  title: Sunset over an underwater forest
+- image: /media/gallery/i3.jpg
+  title: "[i3](https://content.minetest.net/packages/jp/i3/) inventory mod"
 
-- image: /media/gallery/5.jpg
-  title: Cave with a dungeon entrance
+- image: /media/gallery/mineclone2.jpg
+  title: "[Mineclone2](https://content.minetest.net/packages/Wuzzy/mineclone2/) game"
 
-- image: /media/gallery/6.jpg
-  title: Underground cavern with flowing lava
+- image: /media/gallery/mtg_sea.jpg
+  title: An underwater jungle in Minetest Game
```

Comment:
it doesn't make sense to do here because of the odering but IMO one of the screenshot captions could mention that MTG is the default game